Sources/MapboxNavigation/UserLocationStyle.swift

```diff
@@ -0,0 +1,27 @@
+import Foundation
+import MapboxMaps
+
+/**
+ A type that represents a `UIView` that is `CourseUpdatable`.
+ */
+public typealias UserCourseView = UIView & CourseUpdatable
+
+/**
+ The style of user location indicator in a navigation map view.
+ */
+public enum UserLocationStyle {
```

Comment:
Please also update `jazzy.yml` with newly added public symbol.

Reply:
Thanks for the suggestion, I'll update the docs for the `UserLocationStyle `